content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,11 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< warning >}}
+User needs to specify limit in pod spec. If the user doesn't specify limit in pod spec, then resourcequota is not enforced on that resource. So for enforcement of resourcequota on resources, user must specify limit in pod.
```

Comment:
Thanks for documenting it which is helpful in understanding the quota current behaviour. Some suggestion in the sentence:

{{< warning >}}

To make the resource quota work, two things need to be done.
1. admin set the resource quota limit
2. User needs to mention the resource quota limit in pod spec.

If the user doesn't specify the resource limit in pod spec, then the resource quota is not enforced on that resource.
For example, if the admin set the ephemeral-storage quota and if the user doesn't specify the ephemeral-storage limit in the pod creation spec then the ephemeral-storage resource quota will not be enforced. Thus the user can use n number of ephemeral storage.

{{< /warning >}}

Reply:
@gmannos Thank you for the suggestions i have updated the title and comments as you suggested .PTAL
Thanks!

Reply:
> User needs to mention the resource quota limit in pod spec.

If an admission plugin sets default limits for resources, then that defaulting is also adequate. See https://kubernetes.io/docs/concepts/policy/limit-range/ for an example.